apps/cow-fi/COPYRIGHT.md

```diff
@@ -0,0 +1,13 @@
+# Intellectual Property Notice
+
+Copyright 2021 Gnosis Ltd
+
+Copyrights in this project are retained by contributors. No copyright assignment
+is required to contribute to this project.
+
+Except as otherwise noted (below and/or in individual files), this project is
+licensed under the Apache License, Version 2.0
```

Comment:
I know, just copied over.
Still, might be good to have legal review this, as we are no longer in 2021 nor Gnosis